### DISCUSSION TITLE
Tilecache einbinden mit Themenschaltung

### DISCUSSION THREAD

**Discussion by saltdorfer** [e-netze/webgis-community#51] [Q&A]
Wir versuchen gerade langsame Imagedienste als Tilecache einzubinden.

Bei WMTS-Diensten kann man im CMS auch eine Layergruppe hinzufügen und konfigurieren, leider kann man den Layer nur im Themenbaum schalten (eher unpraktisch):
![image](https://user-images.githubusercontent.com/120009698/228203390-9c9de55c-1fc6-4eb8-ab1b-77a77057dd52.png)
nicht aber im Bereich Darstellung:
![image](https://user-images.githubusercontent.com/120009698/228206194-b2541242-e7ec-490f-aa52-47a812a2ebe8.png)

Kommt das noch mit dem nächsten Build? ;)
lg Sabine

**Comment by gis-eni** [comment]
Kommt nicht, sollte schon drin sein ;)

Bei mir schaut das so aus:

<img width="247" alt="image" src="https://user-images.githubusercontent.com/62870568/228743047-9287cdb9-4684-48a8-8727-4ad20d457b0f.png">

Ich geh einmal davon aus, dass du das machen willst? Tiling Dienst über die Darstellungsvarianten schalten?
Bei den Darstellungsvarianten habe ich im CMS ein Container (Tiles) und die entsprechende Darstellungsvariante hinzugefügt:

<img width="863" alt="image" src="https://user-images.githubusercontent.com/62870568/228743369-98a46e91-4e3a-4de8-b25c-fd933a29602c.png">

Die Darstellungsvariante is ``Checkbox`` und verweist auf die Layerschaltung im Dienst

Beim Dienst selber hab ich noch ``Erweiterte Einstellungen`` so bearbeitet, das der Dienst NICHT als Hintergrunddienst behandelt wird, sondern wie ein normaler Dienst:

<img width="358" alt="image" src="https://user-images.githubusercontent.com/62870568/228743897-069a4ce6-f8e1-4af5-b368-d7953dab6228.png">
